src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/yml/workflows.yml

```diff
@@ -69,8 +77,8 @@ framework:
             supports:
                 - Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest
             places:
-                - first
-                - last
+                - { name: first }
+                - { name: last }
```

Comment:
I would even recommend to always use the simple format whenever possible and only switch to the new one when necessary. WDYT?

Reply:
Good Idea 👍

Reply:
I let this test because we need at least one test case for this format.

Reply:
did you keep some tests with the current way? looking at fixtures it feels like you replaced them all?